src/store/user/getters.js

```diff
@@ -0,0 +1,11 @@
+export function getUser(state) {
+  return state.user;
+}
+
+export function isUserAuth(state) {
+  return !!state.user;
```

Comment:
@priyankashrma can you please explain what this mean` !!state.user;`

Reply:
it will convert the object to boolean that mean if user is logged in and there is data in user state it will return true else false.